### ISSUE TITLE
7Span: Accessibility requests approved 11/6/23 (John checking 12/7)

### ISSUE THREAD

**Issue by WendyLBelcher** [PrincetonPEMM/7Span-Phase2#421]
# Last Bugs
### E5: Fix typos
There are typos in the alt text for the main site logo (pricenton, marry), which cause screen readers to output somewhat jumbled audio.
It should be
Princeton Ethiopian, Eritrean, and Egyptian Miracles of Mary (PEMM) project
![image](https://github.com/PrincetonPEMM/7Span-Phase2/assets/56000106/22e1bc7a-c62f-4147-a89c-fb472cd673a7)

### E8: Fix repetition
Original problem: "The mobile menu toggle button has no label or state. Screen reader users just hear "button," rather than "Menu, collapsed, button"
New problem:
- "it now has both which brings us back to the same problem 
- it now reads ""Menu button collapsed, expanded"" when open and 
- ""Menu button close, collapsed"" when closed.

- Also Change the label or use aria-expanded, [not both](https://inclusive-components.design/toggle-button/#changinglabels).
- Also duplicate close button on the menu flyout that has no label.

### E17:  Fix lack of labels and legends
- an unlabeled <button> within the slider that does not appear to have a function for keyboards and should be mapped of the tab order.
- checkboxes do not have legends

# Old

- [ ] Upgrade site to be more accessible.
- [ ] Do NOT make any changes that affect the look, the visible design of the page

Please see the table on accessibility issues.
https://docs.google.com/spreadsheets/d/1-i9ZLkaBO3ZyJoExe4vl8tumUIVBERCkav837iDcLsM/edit#gid=1992549741

Please see Column C, `Area` for instructions. If it is includes the words `7Span`, please implement the changes.
![image](https://github.com/PrincetonPEMM/7Span-Phase2/assets/56000106/f05cac3b-b521-47b9-87d3-efa70c702cda)

We consider some of these to be bugs and have marked them. For instance, the ones about typos and headings. We can discuss if you don't agree on any.
The reviewer who gave us the comments is John Jameson at <jjameson@princeton.edu>. He is very happy to meet with you on Zoom to provide additional instructions, with us or without the PEMM team, as you wish.

**Comment by WendyLBelcher** [comment]
For future
![image](https://github.com/PrincetonPEMM/7Span-Phase2/assets/56000106/437417ab-1c93-40d8-81c1-3e089ec3f749)

**Comment by WendyLBelcher** [comment]
Note, white and yellow is not enough contrast. I'm not sure how to fix it though.
![image](https://github.com/PrincetonPEMM/7Span-Phase2/assets/56000106/d38eef22-dd37-47f6-9bbe-d9695aad6c8b)
